Jenkinsfile

```diff
@@ -626,6 +627,96 @@ pipeline {
                         }
                     }
                 }
+                stage('arm64') {
+                    when {
+                        beforeAgent true
+                        expression { params.arm64 }
+                    }
+                    agent { label 'aarch64' }
+                    environment {
+                        TEST_SKIP_INTEGRATION_CLI = '1'
+                    }
+
+                    stages {
+                        stage("Print info") {
```

Comment:
If these steps are the same as below steps it might make sense to make this a function instead of repeating them all over the place.

Reply:
@seemethere yes! That's something I want to do (not in this PR though); perhaps you could provide input on https://github.com/moby/moby/pull/39708 (where I started playing with doing exactly that?)

In that PR, I wasn't sure how to best move the functions into an include